opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/ProjectsControllerTest.java

```diff
@@ -539,4 +538,8 @@ private void setHandleRenamedFilesToFalse(final String project) {
                 .put(Entity.text(Boolean.FALSE.toString()));
     }
 
+    @Test
+    public void testListFiles() {
+
+    }
```

Comment:
:-D very robust

Reply:
:-)
oops, did not intend to commit this one.